pkgs/tools/misc/nmfd/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "nmfd-${version}";
+  version = "1.1.3";
+  src = fetchFromGitHub {
+    owner = "Ameliorate";
+    repo = "nmfd";
+    rev = "v${version}";
+    sha256 = "1gk3kz9m8547yn9w7cr4a3nlg5pq7z86mksv2ayrmvcicm6m0ifc";
+  };
+
+  makeFlags = [ "INSTALL_PATH=$(out)/bin/" "MANUAL_PATH=$(out)" ];
```

Comment:
```suggestion
  makeFlags = [ "INSTALL_PATH=$(out)/bin/" "MANUAL_PATH=$(out)/share/man/man1" ];
```

Reply:
Thanks, I couldn't figure out how to make manfiles work.